Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clone.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (C) 2023 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+
+from .command import Command
+from .trace import Trace, Relationship
+from webkitbugspy import Tracker, radar
+from webkitscmpy import log
+
+
+class Clone(Command):
+    name = 'clone'
+    help = 'Clone the radar a bugzilla or commit refers to'
+
+    @classmethod
+    def parser(cls, parser, loggers=None):
+        parser.add_argument(
+            'argument', nargs=1,
+            type=str, default=None,
+            help='String representation of a commit to be cherry-picked',
+        )
+        parser.add_argument(
+            '--milestone', '-m',
+            dest='milestone', default=None,
+            help='Milestone to move cloned radar into',
+            type=str,
+        )
+        parser.add_argument(
+            '--why', '-w', '--reason', '-r',
+            dest='reason', default=None,
+            help='Reason why issue is being cloned',
+            type=str,
+        )
+
+    @classmethod
+    def main(cls, args, repository, **kwargs):
+        rdar = None
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, radar.Tracker):
```

Comment:
is this a typo? `rdar` vs `radar`

Reply:
No, might need to change some variable names, `radar` would conflict with `from webkitbugspy import radar`